manifests/server/config.pp

```diff
@@ -224,17 +224,19 @@
       # - $datadir
       # - @extra_systemd_config
 
-    if (versioncmp($facts['puppetversion'], '6.1.0') < 0) {
-      exec { 'restart-systemd':
-        command     => 'systemctl daemon-reload',
-        refreshonly => true,
-        path        => '/bin:/usr/bin:/usr/local/bin',
-      }
-      $systemd_notify = [Exec['restart-systemd'], Class['postgresql::server::service']]
-    }
-    else {
-      $systemd_notify = Class['postgresql::server::service']
+  if (versioncmp($facts['puppetversion'], '6.1.0') < 0) {
+    exec { 'restart-systemd':
+      command     => 'systemctl daemon-reload',
+      refreshonly => true,
+      path        => '/bin:/usr/bin:/usr/local/bin',
+      require     => File['old-systemd-override'],
+      before      => Class['postgresql::server::service'],
     }
+    $systemd_notify = Exec['restart-systemd']
```

Comment:
This change means the files no longer notify the service. Is that intentional?

Reply:
yeah since we added `before` to restart-systemd  resource .
` before      => Class['postgresql::server::service'],`
https://github.com/puppetlabs/puppetlabs-postgresql/blob/8ad345885f1cb7f79601134353d080fdf7fc0c8c/manifests/server/config.pp#L233

Reply:
@ekohl Much appreciated for all the help and review comments. Thank you